staging/src/k8s.io/kubectl/pkg/cmd/create/create_rolebinding_test.go

```diff
@@ -72,7 +73,7 @@ func TestCreateRoleBinding(t *testing.T) {
 	}
 
 	for i, tc := range tests {
-		t.Run(string(i), func(t *testing.T) {
+		t.Run(fmt.Sprintf("%d", i), func(t *testing.T) {
```

Comment:
nit:
```suggestion
		t.Run(strconv.Itoa(i), func(t *testing.T) {
```
`strconv.Itoa` is more efficient in this case.

Reply:
Agreed!, Sprintf will use `reflect` to allocate an extra object.